lib/plugin/allure.js

```diff
@@ -234,6 +234,18 @@ module.exports = (config) => {
     reporter.endCase('passed');
   });
 
+  event.dispatcher.on(event.test.skipped, (test) => {
+    let loaded = true;
+    if (reporter.getCurrentSuite() !== test.parent.fullTitle()) {
```

Comment:
please update to use `if (test.isFastSkipped)` or something like this

Reply:
Added to skip info `test.opts.skipInfo.isFastSkipped`